todo/src/components/TodoList.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { connect } from 'react-redux';
+import { addTodo, toggleTodo } from '../actions';
+
+
+class TodoList extends React.Component {
+    state = {
+        newTodo: ''
+    };
+
+    handleChanges = e => {
+        this.setState({ newTodo: e.target.value })
+    }
+
+    add = e => {
+        e.preventDefault();
+        this.props.addTodo(this.state.newTodo);
+        this.setState({ newTodo: '' })
+    };
+
+    toggleTodo = (e, index) => {
+        e.preventDefault();
+        this.props.toggleTodo(index);
+    }
+
+    render() {
+        return (
+            <React.Fragment>
+                <div className = "to-dos">
+                    {this.props.todos.map((todo, index) => (
```

Comment:
If you console log 'this.props.todos' you get [this](http://prntscr.com/o3mrhr).
It returns 'name' instead of 'todo'!

Reply:
and that's why it wasn't rendering....Thank you!